.travis.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         - cargo fetch --verbose
       script:
         - cargo +nightly-2018-07-17 fmt -- --check && RUST_BACKTRACE=1 cargo test --verbose --all
+        - cargo build && cd test && npm install && npm run start
```

Comment:
Don't we use `yarn` everywhere?

Reply:
In the current configuration, `yarn` is not pre-installed in Travis. I'm looking for a solution.